istioctl/pkg/proxyconfig/proxyconfig.go

```diff
@@ -261,6 +262,17 @@ func printStatus(c *cobra.Command, kubeClient kube.CLIClient, statsType, podName
 			return err
 		}
 		_, _ = fmt.Fprint(c.OutOrStdout(), string(out))
+	case jsonOutput:
+		var data map[string]interface{}
+		if err = json.Unmarshal([]byte(stats), &data); err != nil {
+			return err
+		}
+
+		prettyJSON, err := json.MarshalIndent(data, "", "  ")
```

Comment:
Don't think we need to unmarshal+marshal+indent?

`MarshalIndent` by itself should be fine, or just `Indent`.

Reply:
Curious under what circumstances do you need to use `-ojson` with `istioctl x es`?

In other words,  is `istioctl x es xxxxxxxx | jq` enough?

Reply:
`jq` is fine for querying the resulting JSON, but we shouldn't require the use of `jq` to create human-readable output of a human-computer interface, by default.

Reply:
> Don't think we need to unmarshal+marshal+indent?
> 
> `MarshalIndent` by itself should be fine, or just `Indent`.

Thank you for your suggestion, I have modified it according to your suggestion